.github/workflows/python-install-lint-test.yml

```diff
@@ -61,7 +61,7 @@ jobs:
       if: inputs.mamba_env_name == ''
       run: echo "MAMBAENVNAME=${{ inputs.os }}-3${{ inputs.py3version }}" >> $GITHUB_ENV
 
-    - uses: arup-group/actions-city-modelling-lab/.github/actions/create-mamba-env@main
+    - uses: arup-group/actions-city-modelling-lab/.github/actions/create-mamba-env@test-pin-setup-micromamba
```

Comment:
Don't we want this to be `main`? Is the plan to change it back once this PR is merged?

Reply:
May bad, forgot to revert this one.